sdk/python/tests/integration/online_store/test_universal_online.py

```diff
@@ -443,7 +443,7 @@ def test_online_retrieval_with_event_timestamps(
 
 @pytest.mark.integration
 @pytest.mark.universal_online_stores
-# @pytest.mark.goserver Disabling because the go fs tests are flaking in CI. TODO(achals): uncomment after fixed.
+@pytest.mark.goserver
```

Comment:
not sure if your changes here only fix some of the tests, but there are a few more tests that have this comment